CHANGELOG.md

```diff
@@ -23,6 +23,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Security
 - none yet
 
+## [3.0.0-beta29] - 2017-XX-XX
```

Comment:
FYI, you're not suposed to do that. 👅 
You're supposed to edit the **Unreleased** section, and then when you're releasing you add a new version section with date and everything.
![](https://38.media.tumblr.com/9551302d436dcd825d3512ab563a48b3/tumblr_mii7ujyWAt1rs9keio1_500.gif?h400=&w=600=)